common/device-model.cpp

```diff
@@ -266,7 +266,8 @@ namespace rs2
 
             auto dev_name = get_device_name(dev);
 
-            if( dev.is<update_device>() || is_upgradeable( fw, recommended_fw_ver) )
+            // TODO - D421. Don't suggest to update FW as it doesn't support D421. Revert after 5.17 release with supporting FW
```

Comment:
Can we open tickets for 5.17 major with all the TODO's?

Reply:
OK, I will open tickets.